reporter/slack.go

```diff
@@ -35,9 +35,9 @@ type message struct {
 
 func (w SlackWriter) Write(rs ...models.ScanResult) (err error) {
 
-	channel := w.Cnf.Channel
 	for _, r := range rs {
 		w.lang, w.osFamily = r.Lang, r.Family
+		channel := w.Cnf.Channel
 		if channel == "${servername}" {
```

Comment:
How about checking if w.Cnf.Channel is `${servername}`?
```suggestion
	channel := w.Cnf.Channel
	for _, r := range rs {
		w.lang, w.osFamily = r.Lang, r.Family
		if w.Cnf.Channel == "${servername}" {
```

Reply:
Since channel is only used in the lower for loop block, I closed this suggestion because I thought it should be kept within the scope of the for loop.